docs/ctr-remote.md

```diff
@@ -66,6 +66,18 @@ The following example optimizes an image with a compression level of 1.
 # ctr-remote image optimize --oci --estargz-compression-level 1 ghcr.io/stargz-containers/golang:1.15.3-buster-org registry2:5000/golang:1.15.3-esgz
 ```
 
+You can enable networking inside the container using the `net-host` flag.
```

Comment:
Networking inside container is possible with CNI as well.
e.g. you can use `--cni` option in the demo container([Dockerfile](https://github.com/containerd/stargz-snapshotter/blob/c96a7f9661d70eda53f3f66c09cdbdafcf4ca235/Dockerfile#L187-L197), [compose](https://github.com/containerd/stargz-snapshotter/blob/c96a7f9661d70eda53f3f66c09cdbdafcf4ca235/script/demo/docker-compose.yml)). [`ctr-remote i optimize` is also tested in the container](https://github.com/containerd/stargz-snapshotter/blob/c96a7f9661d70eda53f3f66c09cdbdafcf4ca235/script/optimize/optimize/entrypoint.sh#L260-L331) in our CI.

Reply:
Thank you for your feedback. I'm aware of using CNI plugins to enable networking. I suggested the `net-host` flag as a simpler way to enable networking (similar to `ctr run`) inside the container, since setting up CNI can be more troublesome/error-prone, especially if all you want is basic network connectivity.

Reply:
So I think rather than "You can enable networking inside the container ",  "You can enable *host* networking for the container` seems the right description here.

Reply:
Makes sense. I'll update the docs.

Reply:
Thank you!